### PR TITLE
[LA.UM.5.7.r1] Loire: Enable HZ_1000

### PR DESCRIPTION
**Pull request by bartcubbins**
Signed-off-by: Pavel Dubrova <pashadubrova@gmail.com>

### CODE REVIEW

File: arch/arm64/configs/aosp_loire_kugo_defconfig [hunk 1]
```diff
@@ -43,7 +43,8 @@ CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=6
 CONFIG_QCOM_TLB_EL2_HANDLER=y
 CONFIG_PREEMPT=y
-CONFIG_HZ_100=y
+CONFIG_HZ_1000=y
+CONFIG_HZ=1000
```
Comment (ab123321) [inline arch/arm64/configs/aosp_loire_kugo_defconfig]: This shouldn't be set manually, CONFIG_HZ_1000=y will do the whole job

Comment (artem) [inline arch/arm64/configs/aosp_loire_kugo_defconfig:47]: This shouldn't be set manually, CONFIG_HZ_1000=y will do the whole job

**Comment by kholk** [comment]
It makes sense, but needs a lil testing.

File: arch/arm64/configs/aosp_loire_kugo_defconfig (hunk 1 above)
Comment (bartcubbins) [reply arch/arm64/configs/aosp_loire_kugo_defconfig:47]: @ab123321 then it must be removed from tone and yoshino

**Pull request merged**